### ISSUE TITLE
"import climlab" throws RuntimeError for numpy v1.25 and above

### ISSUE THREAD

**Issue by tatsumonkman47** [climlab/climlab#193]
I have an issue importing climlab using numpy v1.25 and above due to an outdated numpy dependency:

```
>>>import climlab 
RuntimeError: The new DType API is currently in an exploratory phase and should NOT be used for production code.  Expect modifications and crashes!  To experimeiment with the new API you must set `NUMPY_EXPERIMENTAL_DTYPE_API=1` as an environment variable.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/climlab/__init__.py", line 15, in <module>
    from .model.column import GreyRadiationModel, RadiativeConvectiveModel, BandRCModel
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/climlab/model/column.py", line 43, in <module>
    from climlab.radiation.insolation import FixedInsolation
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/climlab/radiation/__init__.py", line 11, in <module>
    from .nband import NbandRadiation, ThreeBandSW
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/climlab/radiation/nband.py", line 4, in <module>
    from climlab.radiation.greygas import GreyGas
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/climlab/radiation/greygas.py", line 5, in <module>
    from climlab.radiation.transmissivity import Transmissivity
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/climlab/radiation/transmissivity.py", line 5, in <module>
    from numpy.core.umath_tests import matrix_multiply
  File "/Users/tatsumonkman/opt/anaconda3/envs/climlab/lib/python3.11/site-packages/numpy/core/umath_tests.py", line 13, in <module>
    from ._umath_tests import *
RuntimeError: cannot load _umath_tests module.
```

I guess the numpy team finally got around to depreciating _umath_tests module this summer. It's not a big issue since I can just roll back to numpy v1.24, but I thought I'd raise it in case you hadn't seen it yet :)

**Comment by adriantompkins** [comment]
Not a long term solution, but as a stop gap you can get climlab working again by downgrading the numpy version, 

`conda install -c conda-forge numpy=1.23`

This gets things going for me in a jupyter notebook, although I then have kernal issues when trying to compile jupyter-book.

**Comment by brian-rose** [comment]
I think the best workaround for now is a dedicated fresh environment with numpy pinned like this:
```
conda create --name myclimlab climlab jupyter python=3.10 numpy=1.24 --channel condo-forge
```

**Comment by adriantompkins** [comment]
... of course in Europe you will need to change that to 

`--channel apartment-forge `

**Comment by adriantompkins** [comment]
More seriously, I would add jupyter-book to that list, as I was going crazy trying to work out why my book wasn't compiling until I realized that jupyter-book is not installed as part of jupyter and as a result I was using the standard install with a different python version:

`conda create --name myclimlab climlab jupyter jupyter-book matplotlib cartopy cdsapi python-cdo python=3.10 numpy=1.24 --channel condo-forge`

**Comment by brian-rose** [comment]
Apparently I completely forgot that this has already been fixed in main branch back in https://github.com/climlab/climlab/commit/7ee9eebd68bccfa458a8244c9d8904fd03423c01. I'll cut a new release shortly so the fix will be available through conda-forge.

**Comment by brian-rose** [comment]
This is now resolved with the release of climlab v0.8.2. 

I did a new clean install with the latest Python and numpy versions with 
```
mamba create --name myclimlab climlab jupyter --channel conda-forge
```
and everything worked as expected.
